src/screens/EditScreen.js

```diff
@@ -1,26 +1,22 @@
-import React, { useState, useContext } from 'react'
-import { View, Text, StyleSheet, TextInput } from 'react-native'
+import React, { useContext } from 'react'
+import { StyleSheet } from 'react-native'
 import { Context } from '../context/BlogContext'
+import BlogPostForm from '../components/BlogPostForm'
 
 const EditScreen = ({ navigation }) => {
-  const { state } = useContext(Context)
+  const { state, editBlogPost } = useContext(Context)
+  const id = navigation.getParam('id')
 
-  const blogPost = state.find(
-    (blogPost) => blogPost.id === navigation.getParam('id')
-  )
-
-  const [title, setTitle] = useState(blogPost.title)
-
-  const [content, setContent] = useState(blogPost.content)
+  const blogPost = state.find((blogPost) => blogPost.id === id)
 
   return (
-    <View>
-      <Text> Edit Title: </Text>
-      <TextInput
-        value={title}
-        onChangeText={(newTitle) => setTitle(newTitle)}
-      />
-    </View>
+    <BlogPostForm
+      initialValues={{ title: blogPost.title, content: blogPost.content }}
```

Comment:
`initialValues`is bringing in the values that were already in place for that blog